ibm_catalog.json

```diff
@@ -0,0 +1,254 @@
+      "long_description": "This solution enables the provisioning of cloud-native application landing zones on IBM Cloud using a range of configurations tailored to different needs — from quick experimentation to enterprise-grade deployments. Each variation offers a distinct balance of automation, integration with IBM Cloud services for security, observability, and compliance, and readiness for evaluation or production use. Whether you're deploying serverless applications on IBM Code Engine or containerized workloads on Red Hat OpenShift, these configurations help accelerate your cloud-native journey. <br/> The Landing zone for cloud-native AI applications includes three core configurations: Quick Start for lightweight experimentation, Basic for flexible serverless deployments on Code Engine with foundational services, and Standard for production-ready OpenShift clusters with advanced security and compliance. Both Basic and Standard also offer variants that include a deployed sample application with watsonx integration and CI/CD pipelines for secure lifecycle management.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-cloud-native-ai/blob/main/README.md",
+      "offering_icon_url": "https://globalcatalog.cloud.ibm.com/api/v1/1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc/artifacts/solution.svg",
+      "provider_name": "IBM",
+      "features": [
+        {
+          "title": "RAG Pattern",
+          "description": "Deploy a RAG pattern with supporting IBM Cloud and watsonx services, and a sample application to Code Engine or Red Hat OpenShift using Continuous Delivery."
+        },
+        {
+          "title": "Implement Security",
+          "description": "The architecture ensures security by deploying Key Protect and Secrets Manager."
+        },
+        {
+          "title": "Achieve Regulatory Compliance",
+          "description": "Ensures regulatory compliance by implementing CI/CD/CC pipelines, along with Security and Compliance Center Workload Protection for continuous compliance."
+        },
+        {
+          "title": "Ensure Observability",
+          "description": "Provides observability by deploying services such as Logs and Monitoring for log analysis and tracking activity."
+        },
+        {
+          "title": "Establish Trust",
+          "description": "Ensures trust by configuring the IBM Cloud account to align with compliance settings as defined in the Financial Services framework."
+        }
+      ],
+      "support_details": "This product is in the community registry, support is handled through the [original repo](https://github.com/terraform-ibm-modules/terraform-ibm-cloud-native-ai). If you experience issues kindly open an issue [here](https://github.com/terraform-ibm-modules/terraform-ibm-cloud-native-ai/issues). Please note that this product is not currently supported through the IBM Cloud Support Center.",
```

Comment:
This is not matching whats in the other repo - we need to use exact same text here